src/libDirectoryService/PoWProcessing.cpp

```diff
@@ -495,6 +512,7 @@ std::set<PubKey> DirectoryService::FindTopPriorityNodes() {
   for (size_t i = 0; i < MAX_SHARD_NODE_NUM && i < vecNodePriority.size();
        ++i) {
     setTopPriorityNodes.insert(vecNodePriority[i].first);
+    lowestPriority = vecNodePriority[i].second;
```

Comment:
Should `lowestPriority` be equal to the smallest priority value inside `vecNodePriority`?
If yes, then we should maybe do this:
1. Outside the loop, set `lowestPriority` to `std::numeric_limits<uint8_t>::min`
2. Inside the loop, set `lowestPriority` to `max(lowestPriority, vecNodePriority[i].second)`

Reply:
vecNodePriority is sorted in descending order, so the last node should have the lowest priority.

Reply:
Ok